### DISCUSSION TITLE
How to disable recording on SD card?

### DISCUSSION THREAD

**Discussion by NoPH8** [roleoroleo/yi-hack-Allwinner-v2#630] [Q&A] [answered]
Hello!
I have Yi Dome Guard YRS3019 on r30gb_0.2.6 which is used as a real-time baby monitor. For this purpose, I need only an Android app and RTSP to have the possibility to watch my baby via VLC from PC.
So I don't need to save records (on an SD card at least) but how can I do it?
I saw the description for "Save video when a motion is detected" that said "(It takes effect only if recording is enabled)" but I can't find where the recording can be disabled.
Thanks!

**Comment by roleoroleo** [comment]
If your cloud is disabled, uncheck this option:
![immagine](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/39277388/05df4ce8-4c02-4130-8aeb-0cf8ad1a207e)

**Reply by NoPH8** [reply]
Thanks for your reply!
But if I disable cloud I'll can't use app, isn't it? =(

**Reply by roleoroleo** [reply]
If you use cloud and the option "Disable cloud" is off, use the app to set the your options.

**Reply by NoPH8** [reply]
There are only 
- Record Detected Activity
- Record Continuous Video

options in the app and no "Do not record to SD card" option.
It is a pity, I hoped that I can do this with the yi-hack.

Anyway thank you for your answers and work!

**Reply by roleoroleo** [reply]
You could try to kill mp4record process at startup.
Inside the startup.sh script.

**Reply by NoPH8** [reply]
Unfortunately the mp4record restarts after `killall mp4record`.
I tried via ssh but I don't think that it have any differences.
It looks for me like that the `cloud` process can monitor the mp4record and starts it again if it fails.

**Reply by NoPH8** [reply, marked as answer]
Thank you for help! Maybe I found the solution of my problem:

- vi /tmp/sd/yi-hack/mp4record
- chmod +x /tmp/sd/yi-hack/mp4record
- mount --bind /tmp/sd/yi-hack/mp4record /home/app/mp4record
- killall mp4record

It needs some time to check that everything is working but now I'm happy =)
